mesh/mesh_readers.cpp

```diff
@@ -1088,44 +1333,172 @@ void Mesh::ReadGmshMesh(std::istream &input, int &curved, int &read_gf)
+                              ho_verts_3D.push_back(hov);
+                              ho_el_order_3D.push_back(el_order);
+                           }
+                           break;
+                        }
+                     case   6: el_order--; //   6-node wedge
+                     case  13: el_order--; //  18-node wedge (2nd order)
+                     case  90: el_order--; //  40-node wedge (3rd order)
+                     case  91: el_order--; //  75-node wedge (4th order)
+                     case 106: el_order--; // 126-node wedge (5th order)
+                     case 107: el_order--; // 196-node wedge (6th order)
+                     case 108: el_order--; // 288-node wedge (7th order)
+                     case 109: el_order--; // 405-node wedge (8th order)
+                     case 110: el_order--; // 550-node wedge (9th order)
+                        {
+                           el_order--; // Gmsh does not define an order 10 wedge
+                           elements_3D.push_back(
+                              new Wedge(&vert_indices[0], phys_domain));
+                           if (el_order > 1)
+                           {
+                              Array<int> * hov = new Array<int>;
+                              hov->Append(&vert_indices[0], n_elem_nodes);
+                              ho_verts_3D.push_back(hov);
+                              ho_el_order_3D.push_back(el_order);
+                           }
+                           break;
+                        }
+                     /*
+                     case   7: el_order--; //   5-node pyramid
+                     case  14: el_order--; //  14-node pyramid (2nd order)
+                     case 118: el_order--; //  30-node pyramid (3rd order)
+                     case 119: el_order--; //  55-node pyramid (4th order)
+                     case 120: el_order--; //  91-node pyramid (5th order)
+                     case 121: el_order--; // 140-node pyramid (6th order)
+                     case 122: el_order--; // 204-node pyramid (7th order)
+                     case 123: el_order--; // 285-node pyramid (8th order)
+                     case 124: el_order--; // 385-node pyramid (9th order)
+                        {
+                           el_order--; // Gmsh does not define an order 10 pyr
+                           elements_3D.push_back(
+                               new Pyramid(&vert_indices[0], phys_domain));
+                           if (el_order > 1)
+                           {
+                              Array<int> * hov = new Array<int>;
+                              hov->Append(&vert_indices[0], n_elem_nodes);
+                              ho_verts_3D.push_back(hov);
+                              ho_el_order_3D.push_back(el_order);
+                           }
+                           break;
+                        }
+                     */
```

Comment:
Do we want to keep the above commented code?

Reply:
Yes, but we should indeed add a comment explaining why it's commented out.